src/node_interface.rs

```diff
@@ -22,17 +22,27 @@ use authority::Authority;
 use types::MessageAction;
 use error::{InterfaceError, ResponseError};
 
+/// MethodCall denotes a specific request to be carried out by routing.
 pub enum MethodCall {
+    /// request for no action
     None,
+    /// request to have `destination` NAE to handle put for the `content`
     Put { destination: NameType, content: Box<Sendable>, },
+    /// request to retreive data of particular type and name from network
     Get { type_id: u64, name: NameType, },
+    /// request to pot
     Post,
+    /// request to update
     Refresh { content: Box<Sendable>, },
+    /// request to send on the request to destination
     SendOn { destination: NameType },
 }
 
+#[deny(missing_docs)]
+/// The Interface trait introduces the methods expected to be implemented by the user
+/// of RoutingNode
 pub trait Interface : Sync + Send {
-    /// the public key or address of the node store it is returned on success.
+    /// the public key or address of the node potentially storing data is returned on success.
```

Comment:
the function return key or address
the function can't be #[deprecated]. stability attributes may not be used outside of the standard library